src/@utils/aquarius/index.ts

```diff
@@ -291,24 +289,17 @@ export async function getPublishedAssets(
   ignoreState = false,
   page?: number,
   type?: string,
-  accessType?: string,
-  complianceType?: string
+  accesType?: string
 ): Promise<PagedAssets> {
   if (!accountId) return
 
   const filters: FilterTerm[] = []
 
   filters.push(getFilterTerm('nft.state', [0, 4, 5]))
   filters.push(getFilterTerm('nft.owner', accountId.toLowerCase()))
-  accessType && filters.push(getFilterTerm('services.type', accessType))
+  accesType !== undefined &&
+    filters.push(getFilterTerm('services.type', accesType))
   type !== undefined && filters.push(getFilterTerm('metadata.type', type))
-  complianceType &&
```

Comment:
This is removing an additional filter present in FMDM

Reply:
Sorry for that, I'll push a fix